lib/message/BuzzerStateMessage.js

```diff
@@ -0,0 +1,75 @@
+var AbstractMessage = require('../AbstractMessage');
+
+/**
+ * This message is from server to client and provides the buzzer state message
+ * or if not set, can be set using the setEnabled method
+ *
+ * @param {Object} args The data for this message object
+ *
+ * @extends AbstractMessage
+ *
+ * @returns {BuzzerStateMessage} An instance
+ */
+var BuzzerStateMessage = function(args) {
+    AbstractMessage.call(this, args);
+    this.set(args);
+};
+
+/**
+ * BuzzerStateMessage is a subclass of AbstractMessage
+ */
+BuzzerStateMessage.prototype = Object.create(AbstractMessage.prototype);
+
+/**
+ * Set the buzzer enabled state for this message object
+ *
+ * @param {Boolean} enabled The state of the buzzer boolean
```

Comment:
Extra `boolean` mentioned - not needed

Generally I don't start a comment like with a capital, as it's usually a continuation of the sentence. For instance, 'Boolean enabled is the state of the buzzer.' or 'Boolean enabled the buzzer state.' etc.